Slingshot.Api/Controllers/ARMController.cs

```diff
@@ -395,6 +396,35 @@ private void DeleteResourceGroup(string subscriptionId, string token, string rgN
             return response;
         }
 
+        [Authorize]
+        [HttpPost]
+        public async Task DeploymentNotification(DeployInputs inputs)
+        {
+            string repositoryUrl = HttpUtility.UrlDecode(inputs.repoUrl);
+            string token = GetTokenFromHeader();
+            Repository repo = Repository.CreateRepositoryObj(repositoryUrl, Request.RequestUri.Host, token);
+            var queryStrings = HttpUtility.ParseQueryString(repositoryUrl);
+            if (queryStrings["pr"] != null)
+            {
+                // if deployment is come from a pull request, post a comment back to the pull request.
+                string siteUrl = string.Format(CultureInfo.InvariantCulture, "https://{0}.azurewebsites.net", inputs.parameters["siteName"]["value"]);
```

Comment:
Ideally, avoid hard coding azurewebsites.net and use same logic as what we already have to give user link to new site